mimic/model/flavor_objects.py

```diff
@@ -1,279 +1,107 @@
 """
-Model objects for mimic flavors.
+Model objects for the flavors.
 """
 
-from characteristic import attributes
+from characteristic import attributes, Attribute
+from json import dumps
+from mimic.model.flavors import (
+    RackspaceStandardFlavor, RackspaceComputeFlavor, RackspaceMemoryFlavor,
+    RackspaceOnMetalFlavor, RackspaceIOFlavor, RackspaceGeneralFlavor,
+    RackspacePerformance1Flavor, RackspacePerformance2Flavor)
```

Comment:
I find it awkward that this module is still named `flavor_objects` and now there's a new module called `flavors`. Perhaps this module could be renamed to `flavor_collections`?